doc/source/reference/random/legacy.rst

```diff
@@ -121,3 +121,63 @@ Distributions
    ~RandomState.wald
    ~RandomState.weibull
    ~RandomState.zipf
+
+Toplevel `numpy.random` functions
```

Comment:
toplevel is kind of an odd word. How about just `Functions in numpy.random`?

Reply:
adopting